Userland/Libraries/LibGfx/ImageFormats/ILBMLoader.cpp

```diff
@@ -189,6 +200,17 @@ static ErrorOr<ByteBuffer> uncompress_byte_run(ReadonlyBytes data, ILBMLoadingCo
     return plane_data;
 }
 
+static ErrorOr<void> extend_ehb_palette(ILBMLoadingContext& context)
+{
+    dbgln_if(ILBM_DEBUG, "need to extend palette");
+    for (size_t i = 0; i < 32; ++i) {
+        auto const color = context.color_table[i];
+        context.color_table.append(color.darkened());
```

Comment:
Same here, you should use `try_append`

Reply:
`try_append`  or `unchecked_append`?

Reply:
To be able to use `unchecked_append` you need to have ensured that the vector can fit all elements you want to add. It's no the case here, so either resize the vector first and use `unchecked_append` or use `try_append`